python/ray/dataframe/datetime.py

```diff
@@ -0,0 +1,33 @@
+import pandas
+import ray
+
+from .dataframe import DataFrame
+from .utils import _map_partitions
+
+
+def to_datetime(arg, errors='raise', dayfirst=False, yearfirst=False, utc=None,
+                box=True, format=None, exact=True, unit=None,
+                infer_datetime_format=False, origin='unix'):
+    if not isinstance(arg, DataFrame):
+        return pandas.to_datetime(arg, errors=errors, dayfirst=dayfirst,
+                                  yearfirst=yearfirst, utc=utc, box=box,
+                                  format=format, exact=exact, unit=unit,
+                                  infer_datetime_format=infer_datetime_format,
+                                  origin=origin)
+    if errors == 'raise':
+        pandas.to_datetime(pandas.DataFrame(columns=arg.columns))
```

Comment:
For the error checking, do you need all of the other parameters?

Reply:
resolved.